modules/virt-hot-plugging-cpu.adoc

```diff
@@ -17,5 +17,10 @@ You can increase or decrease the number of CPU sockets allocated to a virtual ma
 . Select the *vCPU* radio button.
 . Enter the desired number of vCPU sockets and click *Save*.
 +
+[NOTE]
+====
+You can hot-plug up to 3 times the default initial number of vCPU sockets of the VM. Exceeding this limit requires a restart.
```

Comment:
Per [Red Hat Technical Writing Style Guide](https://redhat-documentation.github.io/supplementary-style-guide/#_numbers): Spell out ... numbers zero through nine...
Also, maybe include a bit about not restarting with up to three. 
```suggestion
You can hot-plug up to three times the default initial number of vCPU sockets of the VM without a restart. Exceeding this limit requires a restart.
```